app/models/following.rb

```diff
@@ -1,4 +1,4 @@
 class Following < ApplicationRecord
-  belongs_to :podcast
+  belongs_to :favorite_podcast, foreign_key: 'podcast_id', class_name: 'Podcast'
```

Comment:
Might need to be tested more vigorously. not currently breaking but created an alias